hazelcast/src/main/java/com/hazelcast/spi/impl/sequence/CallIdSequenceWithBackpressure.java

```diff
@@ -40,67 +37,40 @@
  * So perhaps there are a few threads that at the same time see that the there is space and do a next.
  * But any following invocation needs to wait till there is is capacity.
  */
-public final class CallIdSequenceWithBackpressure implements CallIdSequence {
+public final class CallIdSequenceWithBackpressure extends FailFastCallIdSequence {
```

Comment:
I would introduce a shared super class instead of subclassing one implementation by another.

Reply:
OK, I did put a new class AbstractCallIdSequence.